test/grpc/test_replica.js

```diff
@@ -24,10 +24,10 @@ const BASE_UUID = 'c35fa4dd-d527-4b7b-9cf0-436b8bb0ba7';
 const NVMF_URI = /^nvmf:\/\/(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}):\d{1,5}\/nqn.2019-05.io.openebs:/;
 
 // tunables of the test suite
-var endpoint = process.env.MAYASTOR_ENDPOINT;
-var disks = process.env.MAYASTOR_DISKS;
+let endpoint = process.env.MAYASTOR_ENDPOINT;
+let disks = process.env.MAYASTOR_DISKS;
```

Comment:
Should these be documented somewhere?

Reply:
that is a long history. Originally I designed mayastor grpc tests so that they can be run on any mayastor instance later on that proved unfeasible and no one runs the tests that way. Instead, the test suite always starts mayastor and provides necessary resources like disks. So it is a sort of a dead code or dead functionality that would be probably better to remove. But that's out of the scope of this PR.

Reply:
Could you please add a comment stating that?

```suggestion
// TODO: sort of a dead code or dead functionality that would be probably better to remove
// originally made to run tests on any mayastor instance, currently test suite always starts mayastor
// and provides necessary resources like disks
let endpoint = process.env.MAYASTOR_ENDPOINT;
let disks = process.env.MAYASTOR_DISKS;
```

Reply:
fixed, thanks